### ISSUE TITLE
Pegged criteria should only be applicable on FX revenue companies

### ISSUE THREAD

**Issue by CuriousYoda** [dilumn/peggedplacetowork#2]
The '["Great" vs Pegged place to work](https://peggedplacetowork.lk/great_vs_pegged_place_to_work) gives a wrong perspective, when the list contains companies with LKR only revenue. 

The concept of 'USD/EURO pegging' should only be applicable if at least a portion of the revenue is in FX.

**Comment by ahcodes** [comment]
Yeah, agree with @CuriousYoda. Also, we should probably include a list of companies with FX revenue but no active plans to peg the salaries to that particular FX.
